trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/hive/HiveConnectorAdapter.java

```diff
@@ -41,6 +41,7 @@ public Map<String, Object> buildInternalConnectorConfig(GravitinoCatalog catalog
 
     Map<String, Object> properties = new HashMap<>();
     properties.put("hive.metastore.uri", catalog.getRequiredProperty("metastore.uris"));
+    properties.put("hive.security", "allow-all");
```

Comment:
Why does only the hive need these changes?

Reply:
We change the default access Control to use  the internal connector `AccessControl` . However, by default, the Hive connector does not support the 'alter' and 'delete table' operations."

Reply:
What is the relationship between the "Hive connector that does not support the "change" and "delete table" operations and the fact that only the Hive connector requires the modification here?

Reply:
The reference are "https://trino.io/docs/current/connector/hive-security.html#authorization"
We change the code:
```
```
to
```
  @Override
  public ConnectorAccessControl getAccessControl() {
    Connector internalConnector = catalogConnectorContext.getInternalConnector();
    return internalConnector.getAccessControl();
  }
```